acceptance/ui/features/support/hooks.rb

```diff
@@ -7,3 +7,18 @@
   fillInDefaultPassword()
   clickSignInButton()
 end
+
+Before('@emptyHostsPage') do
```

Comment:
Functionally, this use of tags totally works, but I think the tests would read a little better if we converted these hooks to Given steps.  Also, using steps in cases like this would keep more of the feature-related code in the same file (e.g. steps/hosts.rb), making it a little easier for maintenance.

For instance, instead of tagging some Scenarios with @emptyHostsPage, we could start those Scenarios with something like `Given there are no hosts defined`.  Similarly, @defaultHostPage, could be replaced with a step like `Given the default host is defined`